From 86c6fabb6bd28adb85b29fd06de2e92fac4b9fb5 Mon Sep 17 00:00:00 2001
From: "Ai, Ting" <ting.a.ai@intel.com>
Date: Tue, 15 Feb 2022 15:22:47 +0800
Subject: [PATCH] Allow wechat with write permission for 9p share folder

Fix the issue that wechat can not write files in host sharefolder

Signed-off-by: Ai, Ting <ting.a.ai@intel.com>
---
 aafd/logwrapper.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/aafd/logwrapper.te b/aafd/logwrapper.te
index 53433a0..5a705e8 100644
--- a/aafd/logwrapper.te
+++ b/aafd/logwrapper.te
@@ -24,6 +24,7 @@ allow logwrapper p9fs2:dir r_dir_perms;
 
 allow platform_app p9fs2:file r_file_perms;
 allow platform_app p9fs2:dir r_dir_perms;
+allow untrusted_app p9fs2:file rw_file_perms;
 allow untrusted_app_29 p9fs2:file r_file_perms;
 allow untrusted_app_29 p9fs2:file create_file_perms;
 allow untrusted_app_29 p9fs2:dir rw_dir_perms;
-- 
2.25.1

